packages/client/src/components/Question/Question.tsx

```diff
@@ -109,19 +118,23 @@ const Question: FunctionComponent<QuestionProps> = ({
 
       {showQuestionAlert && <QuestionAlert {...{ outcomeType }} />}
 
-      <Nav
-        formEnds={isCheckerConclusive()}
-        nextText={
-          isCheckerConclusive()
-            ? t("outcome.goToOutcome")
-            : t("question.nextQuestion")
-        }
-        showPrev={questionIndex > 0} // Do not show back-button at the first question
-        {...{
-          onGoToPrev,
-          showNext,
-        }}
-      />
+      {!hideNav && (
+        <Nav
+          formEnds={isCheckerConclusive() || isForm}
+          nextText={
+            isForm
+              ? "Ga naar volgende stap"
```

Comment:
Sentence is not in translation.

Reply:
correct, it's one of those things on [the list](https://trello.com/c/pmPRCS6t/2016-one-pager-met-alle-vragen-onder-elkaar)